src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST = 40;
+  const LONG_RENT = 7;
+  const SHORT_RENT = 3;
+  let discount = 0;
+
+  if (days >= LONG_RENT) {
```

Comment:
2. [Checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values. 
You should add constants for long term discount, and short term discount.

and next lines